### PR TITLE
Make tutorials consistent with derivation; fix solutions with arctan2

### PR DESCRIPTION
**Pull request by rikvl**

### CODE REVIEW

**Comment by rikvl** [comment]
@mhvk - This is ready for review.

It should fix the discrepancies we had between our manually generated Earth velocities and those generated by Astropy's `EarthLocation(...).get_gcrs(times).transform_to(lens_frame).velocity.d_z`

It should also resolve the confusion about different v_lens, i_p, and Omega_p solutions. It would be great if you could check if what I did using the `arctan2` functions is correct.

**Comment by rikvl** [comment]
I think I found a solution to the `.npz` file being changed, but it's in the development version of `jupyther-sphinx`.

I think the problem was that different python/numpy/astropy versions (of my environment vs. the one generated by `tox`) gave slightly different numerical values (at the <1e-7 level) in some of the arrays in the `.npz` file.

The new directive `jupyter-input`, added in [this PR](https://github.com/jupyter/jupyter-sphinx/pull/141), allows you to add cells to the notebook that are not executed (but _are_ included in the downloadable `.py` and `.ipynb` files). I tested this solution with the development version of `jupyter-sphinx` and seems to work like a charm. Unless you know of a way of using that development version, I guess we'll have to wait for the [next release](https://github.com/jupyter/jupyter-sphinx/milestone/3) of `jupyter-sphinx`.

For now, I commented out the content of the cell with `np.savez`, and left a remark to remind us to update it when the `jupyter-input` directive becomes available.

**Review by mhvk — APPROVED** [review]
This looks really nice! I've not gone over it in detail, but will merge it, so we can work from it if needed (and also just so that what's in the documentation is correct!)

**Pull request merged**
